### PR TITLE
adding config and button for codesandbox

### PR DESCRIPTION
**Pull request by blackgirlbytes**
I added a button for folks to open this project in Code Sandbox.

### CODE REVIEW

**Comment by blackgirlbytes** [comment]
@acekyd no need to answer now since it's after work hours, but I wanted to double check if adding a tasks.json with npm run build makes sense for this project? 

Thank you!

File: README.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ This project is used as a dependency by several other projects.
 
 Proposals and issues for the specification itself should be submitted as pull requests to the [spec repo](https://github.com/decentralized-identity/decentralized-web-node).
 
+## Running online environment
+
+Interested in contributing instantly? You can make your updates directly without cloning in the running CodeSandbox environment.
+
+[![Edit in CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/github/TBD54566975/dwn-sdk-js/tree/main)
```
[the remark this answers is not in the document]
Comment (blackgirlbytes) [reply README.md:84]: oop. let me double check! thanks for pointing that out.

[the remark this answers is not in the document]
Comment (blackgirlbytes) [reply README.md]: can you try now? @diehuxx

[the remark this answers is not in the document]
Comment (acekyd) [reply README.md]: Works without the `/tree` now

**Review by diehuxx — APPROVED** [review]
looks good. thanks @blackgirlbytes !

**Pull request merged**